src/client/Microsoft.Identity.Client/AppConfig/CacheOptions.cs

```diff
@@ -39,6 +41,19 @@ public CacheOptions(bool useSharedCache)
             UseSharedCache = useSharedCache;
         }
 
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="useSharedCache">Set to true to share the cache between all ClientApplication objects. The cache becomes static. <see cref="UseSharedCache"/> for a detailed description. </param>
+        /// <param name="sizeLimit">Token cache size limit in bytes. <see cref="SizeLimit"/> for a detailed description.</param>
+        public CacheOptions(bool useSharedCache, long sizeLimit)
+        {
+            ValidateSizeLimit(sizeLimit);
+
+            UseSharedCache = useSharedCache;
+            _sizeLimit = sizeLimit;
+        }
+
```

Comment:
This can be written as the following to reuse existing logic:

        public CacheOptions(bool useSharedCache, long sizeLimit) : this(useSharedCache)
        {
            SizeLimit = sizeLimit;
        }